src/config/passport.js

```diff
@@ -0,0 +1,35 @@
+const passport = require("passport");
+const LocalStrategy = require("passport-local").Strategy;
+const sceServiceAPI = require("../services/at-sce-api.service");
+ 
+passport.use(
+    new LocalStrategy({
+            usernameField: "email",
+            passwordField: "password"
+        },
+        async(email, password, done) => {
+            // Match Email's User
+            const request = {
+                email : email,
+                password : password
+            };
+            const user = await sceServiceAPI.login(request);
+            
+            if (!user) {
+                return done(null, false, { message: "Not User found." });
+            } else {
+                console.log(user);
```

Comment:
here we need a second
try this `const user = await sceServiceAPI.getUserById(userAuth.data);`

Reply:
It doesn't works on my local